modules/nf-core/ashlar/ashlar.nf

```diff
@@ -0,0 +1,92 @@
+process wc {
```

Comment:
This module was using the old approach and can be removed, right?

Reply:
You're right.  I've removed it now.